initTerraform/alb.tf

```diff
@@ -0,0 +1,79 @@
+#################### Port 3000 Target Group ####################
+resource "aws_lb_target_group" "frontend_retail_tg" {
+  name        = "D7-bankapp-app"
+  port        = 3000
+  protocol    = "HTTP"
+  target_type = "ip"
+  vpc_id      = aws_vpc.deployment8_vpc.id
+
+  health_check {
+    enabled = true
+    path    = "/health"
+  }
+
+  depends_on = [aws_alb.retail_app]
+}
+
+#################### Port 8000 Target Group ####################
+resource "aws_lb_target_group" "backend_retail_tg" {
```

Comment:
I don't think we need this as the load balancer will be pointing to the front end of the app and won't be connecting directly to the backend